tests/src/JIT/SIMD/Vector3TestNative.cpp

```diff
@@ -247,7 +249,7 @@ EXPORT(void) __stdcall nativeCall_PInvoke_Vector3InComplexStruct(ComplexDT* arg)
     arg->v3.y = arg->v3.y + 1;
     arg->v3.z = arg->v3.z + 1;    
     arg->iv = arg->iv + 1;
-    strncpy(arg->str, ret_str, strnlen("ret_str", 32));
+    snprintf(arg->str, _countof(arg->str), "%s", "ret_string");
```

Comment:
This should be strcpy_s, but we don't have that outside of Windows.  Again, there are implementations in other places; they just aren't currently available to this test.